.travis.yml

```diff
@@ -22,7 +27,7 @@ before_script:
   - if [ "`phpenv version-name`" == "hhvm" ]; then echo 'opcache.enable=1' >> /etc/hhvm/php.ini; fi
   - if [ "`phpenv version-name`" == "hhvm" ]; then echo 'opcache.enable_cli=1' >> /etc/hhvm/php.ini; fi
   - travis_retry composer self-update
-  - travis_retry composer install -o --prefer-source
+  - travis_retry composer install -o --prefer-source $PREFER_LOWEST
```

Comment:
This needs to change to `composer update` please (should've been that before, but this change picked up that fault!